spec/scheduler_spec.rb

```diff
@@ -66,13 +62,6 @@ module Datasets
         scheduler.add
       end
 
-      it "logs each volume that satisfies the filter" do
```

Comment:
If we remove these specs we need to add corresponding specs that it instead returns the volumes that satisfy the filter.